### PR TITLE
Volunteers section issue #146

### PR DESCRIPTION
**Pull request by BaxLi**
1) New volunteer Form done 
2) Save new volunteer via facade
3) small adjustments for new-volunteer.component to comply with Figma
4) Figma icons for select components in new volunteer.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -46,3 +46,9 @@ testem.log
 Thumbs.db
 
 /src/_redirects
+test_login.txt
+angular.json
+package-lock.json
+proxy-local.conf.json
+.gitignore
```
Comment (unrevised6419) [inline .gitignore]: None of these should be ignored

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "singleQuote": true,
+  "printWidth": 120,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "bracketSpacing": true,
+  "semi": false,
+  "useTabs": false,
+  "jsxBracketSameLine": false
+}
```
Comment (unrevised6419) [inline .prettierrc]: Please keep the same file extension `.prettierrc.json` for prettier config

File: angular.json [hunk 3]
```diff
@@ -128,5 +118,8 @@
       }
     }
   },
-  "defaultProject": "client"
+  "defaultProject": "client",
+  "cli": {
+    "analytics": false
+  }
```
Comment (unrevised6419) [inline angular.json]: This was added manually? Or you tried to update angular project?

**Review by unrevised6419 — CHANGES_REQUESTED** [review]

File: angular.json (hunk 3 above)
Comment (BaxLi) [reply angular.json]: no - when start first time the project from fork, Angular asking about permission for catching some analytica data from the project, I'm choose NO ... 
previously  angular.json has only    -  "defaultProject": "client",  
after my answer, it was added by Angular cli (v.11) 
  "cli": {
    "analytics": false
  }

File: .prettierrc (hunk 2 above)
Comment (BaxLi) [reply .prettierrc]: So, as I understand,  .prettier.json now can be changed to 
{
  "htmlWhitespaceSensitivity": "ignore",
  "semi": false,
  "useTabs": false,
  "jsxBracketSameLine": false,
  "overrides": [
    {
      "files": ["*.css", "*.scss"],
      "options": {
        "printWidth": 100
      }
    }
  ]
}

File: .gitignore (hunk 1 above)
Comment (BaxLi) [reply .gitignore]: ok - got it

File: .prettierrc (hunk 2 above)
Comment (unrevised6419) [reply .prettierrc]: Let's keep the defaults for now. Files still are autoformatted.

Comment (unrevised6419) [reply .prettierrc]: No need to make any changes to prettier config files

**Pull request closed without merging**